bundles/org.openhab.binding.ekey/src/main/resources/OH-INF/i18n/ekey_de.properties

```diff
@@ -15,6 +15,8 @@ thing-type.config.ekey.protocol.label = Protokoll
 thing-type.config.ekey.protocol.description = Das vom ekey converter UDP verwendete Protokoll (RARE, MULTI oder HOME, abh�ngig vom verwendeten System).
 thing-type.config.ekey.delimiter.label = Abstandshalter
 thing-type.config.ekey.delimiter.description = Der im ekey converter UDP Protokoll verwendete Feldtrenner.
+thing-type.config.ekey.natIp.label = NAT IP
+thing-type.config.ekey.natIp.description = IPv4 Addresse eines eKey UDP Pakets. Kann abweichend von der ipAddress sein wenn NAT genutzt wird (z.B. in Kubernetes) 
```

Comment:
Please remove these changes from the German translations. This is to be done later through Crowdin.